README.md

```diff
@@ -450,6 +452,51 @@ else {
 }
 ```
 
+## `OIDC` token.parseAndStoreTokensFromUrl(tokenStorageKeys)
+
+Parse and **store** the access or ID Tokens from the url after a successful authentication redirect. This is used when `authParams.display = 'page'`.
+
+* `tokenStorageKeys` - Override the default storage naming conventions for the access and/or ID tokens. Defaults to `accessToken` and `idToken`.
+
+```javascript
+var signIn = new OktaSignIn({
+  baseUrl: 'https://{yourOktaDomain}',
+  clientId: '{{myClientId}}',
+  redirectUri: '{{redirectUri configured in OIDC app}}',
+  authParams: {
+    responseType: ['id_token', 'token'],
+    // `display: page` will initiate the OAuth2 page redirect flow
+    display: 'page'
+  }
+});
+
+// The user has just landed on our login form, and has not yet authenticated
+if (!signIn.token.hasTokensInUrl()) {
+  signIn.renderEl({el: '#osw-container'});
+}
+
+// The user has redirected back after authenticating and has their access and/or
+// ID Token in the URL.
+else {
+  signIn.token.parseAndStoreTokensFromUrl()
+  .then(function() {
+    const idToken = signIn.tokenManager.get('idToken');
+    console.log(idToken);
+  });
+
+  // Optionally - use a different key for the tokens
+  signIn.token.parseAndStoreTokensFromUrl({
+    ACCESS_TOKEN: 'access-token-key',
```

Comment:
If I want to store things in the tokenManager with my own keys, why not just use `parseTokensFromUrl`? I think `parseAndStoreTokensFromUrl` should be opinionated.

Reply:
Developers would need to re-implement logic that we're already providing. By extending this method to optionally take in keys for the tokens, they'd be equipped to:

1. Support multiple tokens of a single type
2. Namespace tokens